src/pages/guides/using/commerce-custom-events.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Publish your own Custom Events with Adobe Commerce
+---
+
+# Publish your own Custom Events with Adobe Commerce
+
+These instruction describe how to set up and get started with Adobe I/O Events for Adobe Commerce. You can use Adobe I/O for sending Adobe Commerce user driven events.
+
+## Introduction
+
+This document will describe all steps necessary to setup, emit and receive events. Firstly, the event provdire will be set up, then we will trigger events from Adobe Commerce and finally an event registration will be created.
+
+## Requirements
+
+You will need for this tutorial:
```

Comment:
`magento-cloud` CLI tool is also needed. We should mention it and provide a link to https://devdocs.magento.com/cloud/reference/cli-ref-topic.html